shopify/limits.py

```diff
@@ -1,31 +1,61 @@
 import shopify
 
+
 class Limits(object):
-    CREDIT_LIMIT_HEADER_PARAM = 'x-shopify-shop-api-call-limit'
+    """
+    API Calls Limit
+    https://help.shopify.com/en/api/getting-started/api-call-limit
 
-    @classmethod
-    def credit_left(cls):
-        return cls.credit_limit() - cls.credit_used()
+    Conversion of lib/shopify_api/limits.rb
+    """
+    # num_requests_executed/max_requests
+    # Eg: 1/40
+    CREDIT_LIMIT_HEADER_PARAM = 'http_x_shopify_shop_api_call_limit'
```

Comment:
i’m confused why this switches to underscores from dashes

Reply:
You're right, I updated this to be exactly like the ruby implementation: https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/limits.rb#L14

Reply:
In Ruby land, Rack transforms the headers to underscores